README.md

```diff
@@ -59,6 +59,25 @@ import { getAuthenticatorFromEnvironment } from 'ibm-cloud-sdk-core';
 const iamAuthenticator = getAuthenticatorFromEnvironment('my-service');
 ```
 
+## Logging
+This package uses [debug](https://www.npmjs.com/package/debug) for logging.
+
+- Logging is disabled by default.
+- In order to see the log output, set the environment variable ``DEBUG`` including the desired scope.
+  - ```DEBUG=ibm-cloud-sdk-core:error``` enables error logs
+  - ```DEBUG=ibm-cloud-sdk-core:info``` enables info logs
+  - ```DEBUG=ibm-cloud-sdk-core:debug``` enables debug logs
+  - ```DEBUG=ibm-cloud-sdk-core:verbose``` enables verbose logs
+  - ```DEBUG=ibm-cloud-sdk-core:warning``` enables warning logs
```

Comment:
I suppose this is fine but I feel like the typical idea in loggers is that you set _one_ level and then you get the logs from that level and also everything "under" that level in terms of decreasing verbosity. For example, if I set the level "verbose", I would also expect to get error and warnings and info, etc.

This seems like a limitation of `debug` and not your implementation so I'm not sure there's anything we can do here but I hope we aren't creating an experience that's less than idea

Reply:
This wasn't specified in the documentation, and yes it's a limitation of using debug. We can manually force this ourselves by enabling each log level through some conditions, but something to keep in mind is that I noticed debug was being used by other packages in our code. So that might make our version of log levels different than what the other packages do if they did not manually enforce that.

Reply:
I noticed the same thing and would like to put in my vote for making the levels "additive", with a clear indication of the order of inclusion.

Reply:
Sounds good. I've gone ahead and implemented log levels